docs/deploying.md

```diff
@@ -167,15 +166,24 @@ Resources:
       LaunchTemplateData:
         ImageId: !Ref FckNatAMIParameter
         InstanceType: t4g.nano
+        NetworkInterfaces:
+          - DeviceIndex: 0
+            AssociatePublicIpAddress: true
+            Groups:
+            - !GetAtt [FckNatSecurityGroup, GroupId]
         IamInstanceProfile:
           Name: !Ref FckNatAsgInstanceProfile
-        SecurityGroupIds:
-          - !GetAtt [FckNatSecurityGroup, GroupId]
         UserData:
-          Fn::Base64: !Sub |
-            #!/bin/bash
-            echo "eni_id=${FckNatInterface}" >> /etc/fck-nat.conf
-            service fck-nat restart
+          Fn::Base64:
+            Fn::Join:
+              - ""
+              - - |-
+                  #!/bin/bash
+                  echo "eni_id=
+                - !Ref FckNatInterface
+                - |-
+                  " >> /etc/fck-nat.conf
+                  service fck-nat restart
```

Comment:
I don't think there were any issues with the previous version of this? I would much prefer to keep this using `!Sub`, it's a lot cleaner.

Reply:
Yeah sorry about that, I've reverted this change